src/main/java/core/basesyntax/Main.java

```diff
@@ -0,0 +1,44 @@
+package core.basesyntax;
+
+import core.basesyntax.model.Transaction;
+import core.basesyntax.service.Writer;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderImpl;
+import core.basesyntax.service.impl.ReportServiceImpl;
+import core.basesyntax.service.impl.WriterImpl;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.OperationStrategy;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Main {
+    private static final String READ_PATH = "src/main/java/fruits";
+    private static final String WRITE_PATH = "src/main/java/core/basesyntax/newFIle";
```

Comment:
https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop#place-the-input-and-output-files-into-the-srcmainresources-folder

your files should be located in src/main/resources folder and end with `.csv`